src/frontend-scripts/components/App.jsx

```diff
@@ -164,6 +164,14 @@ export class App extends React.Component {
 			});
 		});
 
+		socket.on('feedbackReceived', () => {
```

Comment:
I think this would be better as a  `feedbackResponse` emit with a status code passed in as the param, just to make it a bit less verbose.